pkg/functions/client.go

```diff
@@ -618,14 +618,61 @@ func (c *Client) Build(ctx context.Context, f Function) (Function, error) {
 
 	// TODO: create a status structure and return it here for optional
 	// use by the cli for user echo (rather than rely on verbose mode here)
-	message := fmt.Sprintf("🙌 Function image built: %v", f.Image)
+	message := fmt.Sprintf("🙌 Function built: %v", f.Image)
 	if runtime.GOOS == "windows" {
-		message = fmt.Sprintf("Function image built: %v", f.Image)
+		message = fmt.Sprintf("Function built: %v", f.Image)
 	}
 	c.progressListener.Increment(message)
 	return f, err
 }
 
+// Scaffold writes a functions's scaffolding to a given path.
+// It also updates the included symlink to function source 'f' to point to
+// the current function's source.
+func (c *Client) Scaffold(ctx context.Context, f Function, dest string) (err error) {
```

Comment:
This method will actually be refactored to instead invoke scaffolding from its own package in a forthcoming commit such that the oci builder can invoke it directly without creating a cyclical dependency.